deploy-manage/deploy/cloud-enterprise/ece-wildcard-dns.md

```diff
@@ -18,3 +20,5 @@ The `ip.es.io` service is provided to help you evaluate {{ece}} without having t
 
 A wildcard certificate is enabled based on the deployment domain name. For more information on modifying the deployment domain name, check [Configure endpoints](change-endpoint-urls.md). The deployment domain name also determines the endpoint URLs that are displayed in the Cloud UI.
 
+Report security issues to security@elastic.co. 
```

Comment:
we don't have automatic url rewrite on these so I think we need to format it as a link so it looks right

```suggestion
Report security issues to [security@elastic.co](mailto:security@elastic.co). 
```